test/categories.test.js

```diff
@@ -0,0 +1,166 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../api/app';
+import { Category, allCategories } from '../api/v1/models/Category';
+
+const should = chai.should();
+chai.use(chaiHttp);
+
+const initialCategory = new Category({
```

Comment:
Remember that this data should ideally not be inside the test. You should create a separate file for it and import it here.

Reply:
Okay. I'll do so